docs/outlook/compare-outlook-add-in-support-in-outlook-for-mac.md

```diff
@@ -18,7 +18,7 @@ For information about new UI support, see [Add-in support in Outlook on new Mac
 | Supported versions of office.js| All APIs in Office.js. | All APIs in Office.js.<br><br>**NOTE**: In Outlook on Mac, only build 16.35.308 or later supports saving a meeting. Otherwise, the `saveAsync` method fails when called from a meeting in compose mode. See [Cannot save a meeting as a draft in Outlook for Mac by using Office JS API](https://support.microsoft.com/help/4505745) for a workaround. |
 | Instances of a recurring appointment series | <ul><li>Can get the item ID and other properties of a master appointment or appointment instance of a recurring series.</li><li>Can use [mailbox.displayAppointmentForm](/javascript/api/requirement-sets/outlook/preview-requirement-set/office.context.mailbox#methods) to display an instance or the master of a recurring series.</li></ul> | <ul><li>Can get the item ID and other properties of the master appointment, but not those of an instance of a recurring series.</li><li>Can display the master appointment of a recurring series. Without the item ID, cannot display an instance of a recurring series.</li></ul> |
 | Recipient type of an appointment attendee | Can use [EmailAddressDetails.recipientType](/javascript/api/outlook/office.emailaddressdetails#outlook-office-emailaddressdetails-recipienttype-member) to identify the recipient type of an attendee. | `EmailAddressDetails.recipientType` returns `undefined` for appointment attendees. |
-| Version string of the client application | The format of the version string returned by [diagnostics.hostVersion](/javascript/api/outlook/office.diagnostics#outlook-office-diagnostics-hostversion-member) depends on the actual type of client. For example:<ul><li>Outlook on Windows: `15.0.4454.1002`</li><li>Outlook on the web and new Outlook on Windows (preview): `15.0.918.2`</li></ul> |An example of the version string returned by `Diagnostics.hostVersion` on Outlook on Mac: `15.0 (140325)` |
+| Version string of the client application | The format of the version string returned by [diagnostics.hostVersion](/javascript/api/outlook/office.diagnostics#outlook-office-diagnostics-hostversion-member) depends on the actual type of client. For example:<ul><li>Outlook on Windows: `16.0.4454.1002`</li><li>Outlook on the web and new Outlook on Windows (preview): `16.0.918.2`</li></ul> |An example of the version string returned by `Diagnostics.hostVersion` on Outlook on Mac: `16.0 (140325)` |
```

Comment:
```suggestion
| Version string of the client application | The format of the version string returned by [diagnostics.hostVersion](/javascript/api/outlook/office.diagnostics#outlook-office-diagnostics-hostversion-member) depends on the Outlook client. <ul><li>Outlook on Windows: Returns the version of the Outlook client. For example, `16.0.4454.1002`.</li><li>Outlook on the web and new Outlook on Windows (preview): Returns the version of the Exchange Server. For example, `15.0.918.2`</li></ul> |The `diagnostics.hostVersion` call returns the version of the Outlook client. For example, `16.0 (140325)`.|
```
The Exchange Server version still starts with 15 (see https://learn.microsoft.com/exchange/new-features/new-features?view=exchserver-2019).